src/vrl/api/tests/test_selectielijst.py

```diff
@@ -54,3 +56,21 @@ def test_filter_procestype(self):
             response_data['results'][0]['procesType'],
             f'http://testserver{procestype_url}',
         )
+
+
+class ResultaatTests(APITestCase):
+
+    def test_get_resultaat(self):
+        """ test resultaat get api:
```

Comment:
We tend to format docstrings in the following way:

```python
def foo():
    """
    Foo as you like.

    This is a more elaborate description of what foo does and requires.
    """
```
See also [PEP257](https://www.python.org/dev/peps/pep-0257/)